blob/service.go

```diff
@@ -131,6 +132,8 @@ func (s *Service) GetAll(ctx context.Context, height uint64, namespaces []share.
 				resultErr[i] = fmt.Errorf("getting blobs for namespace(%s): %s", namespace.String(), err)
 				return
 			}
+
+			log.Debugw("receiving blobs", "height", height, "total", len(blobs), "blobs", blobs)
```

Comment:
I'd not print the blobs as that would cloud logs, the length is sufficient

Reply:
Removed `blobs`

Reply:
BWT, by removing `blobs` we lost the possibility to map for which namespace the blobs are received.
```
2023-09-28T10:14:51.472+0300	DEBUG	blob	blob/service.go:136	receiving blobs	{"height": 118249, "total": 1}
```